Bottom: 5e31d18740fdf716c874a21e313147d406daa00c
Top:    6d1c30406729b36455e0d80d169c07318964c5c4
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-10-22 00:33:23 +0100

Hack around Circuit CAM / EDG brokeness reading drill files







---

diff --git a/src/hid/gerber/gerber.c b/src/hid/gerber/gerber.c
index e95186b..54c010f 100644
--- a/src/hid/gerber/gerber.c
+++ b/src/hid/gerber/gerber.c
@@ -462,10 +462,17 @@ gerber_set_layer (const char *name, int group)
 	      int ap = findApertureCode (pending_drills[i].diam, ROUND);
 	      fprintf (f, "T%02d\015\012", ap);
 	    }
+#ifdef EDG_ARE_BROKEN
+	  fprintf (f, "X%06ldY%06ld\015\012",
+		   100 * gerberX (PCB, pending_drills[i].x), 100 * gerberY (PCB,
+								pending_drills
+								[i].y));
+#else
 	  fprintf (f, "X%06ldY%06ld\015\012",
 		   gerberX (PCB, pending_drills[i].x), gerberY (PCB,
 								pending_drills
 								[i].y));
+#endif
 	}
       free (pending_drills);
       n_pending_drills = max_pending_drills = 0;
